new-backoffice/README.md

```diff
@@ -8,7 +8,7 @@ description: >-
 
 Version 13 of Umbraco CMS will include a new backoffice. It will look more or less the same but be built on modern technology using a UI Library built for the purpose.
```

Comment:
```suggestion
Version 14 of Umbraco CMS will include a new backoffice. It will look more or less the same but be built on modern technology using a UI Library built for the purpose.
```